marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/controllers/MappingController.java

```diff
@@ -83,4 +73,23 @@ protected MappingService getMappingService() {
         @ApiModelProperty("Each property object has a name matching that of an entity property and a sourceFrom mapping expression")
         public Map<String, Object> properties;
     }
+
+
+    public static class RelatedEntityMapping {
+        public String mappingLinkText;
+        public String entityMappingId;
+    }
+
+    public static class EntityForUI {
```

Comment:
How about "MappableEntity"? I think that better implies what this is - the entity model plus a few mapping-specific bits of data. The method name would then be "getMappableEntities".